R/data-importer-configuration.R

```diff
@@ -239,28 +238,31 @@ DataImporterConfiguration <- R6::R6Class(
     }
   ),
   public = list(
-    #' @description
-    #' Initialize a new instance of the class
-    #' @param configurationFilePath Path to the XML file with stored configuration
-    #' (e.g. create in PK-Sim or MoBi).
+    #' @param ref Reference to .NET DataImporterConfiguration object
     #' If `NULL` (default), an empty configuration with columns "Time" and
     #' "Measurement" is created.
+    #' @description
+    #' Initialize a new instance of the class
     #' @return A new `DataImporterConfiguration` object.
-    initialize = function(configurationFilePath = NULL) {
+    initialize = function(ref = NULL) {
```

Comment:
No "either or", only `ref`. For creating from file, there is a function.

Reply:
good!